README.md

```diff
@@ -4,18 +4,29 @@ A java based file attribute caching filesystem library
 This library is meant to wrap another filesystem in order to intercept and cache calls for reading and setting file
 attributes.
 
-It reduces OTHER_IOPS calls as listed in the "other" column when executing [Procmon] and looking at
+It reduces `OTHER_IOPS` calls as listed in the "other" column when executing [Procmon] and looking at
 operations performed on a given file where file attributes are read and/or set.
 
 ## Usage Instructions
-Dependency to add via gradle:
+Ensure your `settings.gradle.kts` file can pull dependencies from maven central.
+```kotlin
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+    }
+}
+```
+
+Dependency to add to your top level `build.gradle.kts` file:
 ```kotlin
 dependencies {
+    // If you want to use the attribute caching filesystem for both running code and test code use it this way.
     implementation("com.pkware.filesystem:file-attribute-caching:VERSION_HERE")
```

Comment:
sorry im not understanding what you are meaning here? I included both lines and the comments say to use one or the other depending on if you need the caching for testing and runnable code or just testing

Reply:
ahh we talked about this and Im going to leave it as is since i have a comment about changing the version and VERSION_HERE will prompt the user to change the version to the most updated one rather than having the dev update it each time we do a new release